### PR TITLE
remove busybox and add securityContext for streamsets container

### PR DESCRIPTION
**Pull request by dariakharlan**

### CODE REVIEW

File: charts/streamsets/templates/statefulset.yaml [hunk 1]
```diff
@@ -20,14 +20,6 @@ spec:
         app.kubernetes.io/name: {{ include "streamsets.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
-      initContainers:
```
Comment (kiamran) [inline charts/streamsets/templates/statefulset.yaml:23]: @romanki do we need this for onprem ?

**Comment by romanki** [comment]
don't remember exactly what it was for, maybe we did some small simple config for start script with init container (something related to bc on times when we started that integration), maybe now it is not relevant.

File: charts/streamsets/templates/statefulset.yaml (hunk 1 above)
Comment (romanki) [reply charts/streamsets/templates/statefulset.yaml:23]: ah i see in change it did set some permissions, but not sure if it relevant for onprem, we will check it with the next version.

**Comment by dariakharlan** [comment]
Thanks Roman, I'll add an option to enable it just in case it's still needed

**Pull request merged**